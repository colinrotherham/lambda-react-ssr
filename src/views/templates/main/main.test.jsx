import React from 'react'
import renderer from 'react-test-renderer'
import Template from './main'

describe('Template: Main', () => {
  test('Component is rendered', () => {
    const component = renderer.create(<Template>Hello World</Template>)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
