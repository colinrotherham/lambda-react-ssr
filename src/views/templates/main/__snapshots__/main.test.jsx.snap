// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template: Main Component is rendered 1`] = `
<html
  lang="en-GB"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <title>
      
    </title>
    <meta
      content="width=device-width, initial-scale=1, viewport-fit=cover"
      name="viewport"
    />
  </head>
  <body>
    Hello World
  </body>
</html>
`;
